### PR TITLE
Add unit role to data

### PR DESCRIPTION
**Pull request by stelar7**
Havent tested this yet, but should work in theory

### CODE REVIEW

File: cdtb/tftdata.py [hunk 1]
```diff
@@ -261,6 +262,7 @@ def parse_champs(self, map22, traits):
 
             tft_bin = BinFile(self_path)
             record = next((x for x in tft_bin.entries if x.type == "TFTCharacterRecord"), {})
+
```
Comment (benoitryder) [inline cdtb/tftdata.py:265]: Unneeded change

File: cdtb/tftdata.py [hunk 2]
```diff
@@ -291,6 +293,11 @@ def parse_champs(self, map22, traits):
                 rarity = champ.getv("mRarity", 0) + 1
                 cost = rarity + int(rarity / 6)
 
+            role = "Unknown Role"
+            roleKey = record.getv("CharacterRole", None)
```
Comment (benoitryder) [inline cdtb/tftdata.py:297]:
```suggestion
            role_key = record.getv("CharacterRole", None)
```

File: cdtb/tftdata.py [hunk 3]
```diff
@@ -291,6 +293,11 @@ def parse_champs(self, map22, traits):
                 rarity = champ.getv("mRarity", 0) + 1
                 cost = rarity + int(rarity / 6)
 
+            role = "Unknown Role"
```
Comment (benoitryder) [inline cdtb/tftdata.py:296]: I see no reason for this fallback/default value. We should just use `None`.

**Review by benoitryder — CHANGES_REQUESTED** [review]

**Comment by Morilli** [comment]
I wonder if it could be worth exporting the whole `TFTCharacterRoleData` information as well...

**Comment by benoitryder** [comment]
> I wonder if it could be worth exporting the whole `TFTCharacterRoleData` information as well...

We can start with just the name, then add roles data in a separate top-level entry.

**Comment by benoitryder** [comment]
Replaced by #106

**Pull request closed without merging**